python/lzreposync/src/lzreposync/deb_metadata_parser.py

```diff
@@ -0,0 +1,70 @@
+#  pylint: disable=missing-module-docstring
+
+import logging
+import sys
+
+from lzreposync.packages_parser import PackagesParser
+from lzreposync.translation_parser import TranslationParser
+from spacewalk.satellite_tools.syncLib import log, log2
```

Comment:
Just a heads-up: logging in reposync currently suffers from a split-brain problem. We have `rhnLog` (with `syncLib` defining wrapper functions) one the one hand and `logging` on the other. The latter is also used by 3rd-party libraries (e.g. `requests`). It's one of the things on our list to clean up, so it does not matter much what you use now. That said, I suggest to stick to one of them to make it easy for yourself.